plans/stabilization.fmf

```diff
@@ -0,0 +1,8 @@
+summary: Pre-release "stabilization" testing
+discover:
+    how: fmf
+    filter:
+      - tag:-adhoc
+      - tag:-broken
+environment:
+    CONTEST_CONTENT_BRANCH: master
```

Comment:
Stabilization is done on stabilization branch. Unfortunately, stabilization branch doesn't have static branch name as it has version suffix `stabilization-v0.1.X`.

Reply:
Right, I'm not even sure if I want to keep `CONTEST_CONTENT_BRANCH: master` here if it should be user-specified on TMT CLI (via runcontest or otherwise). ... And if we can't embed it for stabilization, maybe that's another reason to not have it in the plans.